js/main.js

```diff
@@ -203,32 +203,34 @@ function levelRatio(fertilizer, level, isWildseed) {
 }
 
 /*
- * Calculates the keg modifier for the crop.
+ * Calculates the keg base price for the crop.
  * @param crop The crop object, containing all the crop data.
- * @return The keg modifier.
+ * @return The keg base price.
  */
-function getKegModifier(crop) {
-	if (crop.produce.kegType == "Coffee" || crop.produce.kegType == "Tea"){
-		result = 1;
-	} else {
-		result = crop.produce.kegType == "Wine" ? 3 : 2.25;
-	}
-	
-    return result;
+function getKegBasePrice(crop) {
+
+	return crop.produce.keg != null ? crop.produce.keg : crop.produce.kegType == "Wine" ? crop.produce.price * 3 : crop.produce.price * 2.25;
 }
 
 /*
  * Calculates the cask modifier for the crop.
  * @param crop The crop object, containing all the crop data.
  * @return The cask modifier.
  */
-function getCaskModifier() {
-    switch (options.aging) {
-        case 1: return 1.25;
-        case 2: return 1.5;
-        case 3: return 2;
-        default: return 1;
-    }
+function getCaskModifier(crop) {
+	var modifier = 1
+	if (options.aging > 0 && crop.produce.ages){
+		switch (options.aging) {
+			case 1: modifier = 1.25;
+				break;
+			case 2: modifier = 1.5;
+				break;
+			case 3: modifier = 2;
+				break;
+			default: modifier = 1;
```

Comment:
Again, nitpicking, but you could just break on the default case since modifier is initialized at 1.
I would advise against removing the case entirely since it can still catch unexpected values and just for future-proofing sake.